### DISCUSSION TITLE
how to check request body content

### DISCUSSION THREAD

**Discussion by techieshark** [hyperjumptech/monika#649] [Q&A] [answered]
Hi, 

Is it possible to set up a probe which declares that certain content exists on a page, e.g. that the html's `<title>` matches some expected value?

Thanks!

**Comment by nicnocquee** [comment, marked as answer]
Hello @techieshark 

You can use Regex for the alert's query. For example, if you want to receive notification when the title is "Error not found", you can use the following alert:

```yaml
 - query: response.body ~= "Error not found"
        message: Title not what is expected!
```

Likewise, If you want to get notification when the title does not contain "Where the world", you can use the following alert

```yml
 - query: response.body ~= "^(?![^]*Where the world)"
        message: Title not what is expected!
```

**Reply by techieshark** [reply]
Thanks @nicnocquee!
